packages/doctype-tile/src/__tests__/tile-doctype.test.ts

```diff
@@ -267,6 +267,41 @@ describe('TileDoctypeHandler', () => {
     expect(state).toMatchSnapshot()
   })
 
+  it.only('squashes record correctly', async () => {
```

Comment:
I'm not the most familiar with jest, but from a quick google it seems like using `it.only` here will cause the other tests in this file not to run?  @oed

Reply:
err, yeah this should have been removed -.-

Reply:
It's useful when you just want to run a test while coding.

Reply:
kinda thing we could have a lint rule for, once we have a linting system in place